acm/src/main/java/org/literacybridge/acm/store/Committable.java

```diff
@@ -23,17 +25,14 @@
     private CommitState commitState;
     private DeleteState deleteState;
 
-    private CommitListener listener;
+    // remembers whether this is a new Committable that was added to the index between the doCommit() and doAfterCommit() steps
```

Comment:
Changed the comment:
    // remembers whether this is a new object that was added to the index, or whether an existing object was updated

Does it make sense now?